Makefile

```diff
@@ -30,6 +30,7 @@ UNFORMATTED=$(shell gofmt -l .)
 
 GET_TOOL=go get -modfile=tool.mod -tool
 RUN_TOOL=go tool -modfile=tool.mod
+SSVSIGNER_RUN_TOOL=go tool -modfile=../tool.mod
```

Comment:
This should be `go run` instead of `go tool`:

```makefile
  SSVSIGNER_RUN_TOOL=go run -modfile=../tool.mod
```
go tool is for built-in Go tools, while go run is needed to execute packages from tool.mod like golangci-lint.